resources/build/utils.inc.sh

```diff
@@ -214,6 +126,7 @@ _try_multiple_times ( ) {
 # Verifies that node is installed, and installs npm packages, but only once per
 # build invocation
 #
+# TODO: rename to builder_node_select_version_and_npm_ci, move to builder.node.inc.sh
```

Comment:
TODO:
* rename `verify_npm_setup` to `builder_node_select_version_and_npm_ci`, move to builder.node.inc.sh
* rename `_print_expected_node_version` to `_node_print_expected_version`, move to builder.node.inc.sh
* rename `_select_node_version_with_nvm` to `_node_select_version_with_nvm`, move to builder.node.inc.sh